visualization/__init__.py

```diff
@@ -0,0 +1,3 @@
+from . import plot_annot_properties as visannot
+from . import plot_contours as viscontours
+from . import utils as utils
```

Comment:
This is a fine way to organize your `__init__.py` file, but I would just name the modules visannot.py, viscontours.py, and utils.py. You can still import them, but people will be surprised if they have two names.